finetune/base_models/bert/featurizer.py

```diff
@@ -325,20 +358,22 @@ def chunk_aggregation(pooled_outputs, pad_mask):
 
         TODO Implement LSTM
 
-        TODO Add sequence mask for chunks that are all 0 (pad)
+        TODO Fix attention and add pad_mask
         """
+        inv_pad_mask = tf.cast(1 - pad_mask, dtype=pooled_outputs.dtype)
         if config.chunk_pool_fn == "mean":
             # aggr_outputs = tf.reduce_mean(pooled_outputs, axis=1)
-            inv_pad_mask = tf.cast(1 - pad_mask, dtype=pooled_outputs.dtype)
             denom = tf.reduce_sum(inv_pad_mask, axis=1)
             aggr_outputs = tf.reduce_sum(pooled_outputs * inv_pad_mask, axis=1) / denom
         elif config.chunk_pool_fn == "attention":
-            aggr_outputs = attn(pooled_outputs, config.weight_stddev)
+            # aggr_outputs = attn(pooled_outputs, config.weight_stddev)
+            aggr_outputs = attn(pooled_outputs, config)
         elif config.chunk_pool_fn == "max":
-            aggr_outputs = tf.reduce_max(pooled_outputs, axis=1)
+            aggr_outputs = tf.reduce_max(pooled_outputs * inv_pad_mask, axis=1)
```

Comment:
this makes padded values 0. But because we have positive and negative values this should be `- pad_max * <large number>`